packages/ui/src/theme/__tests__/defaultTheme.test.ts

```diff
@@ -543,7 +547,12 @@ describe('@aws-amplify/ui', () => {
         --amplify-components-select-min-width: 6.5rem;
         --amplify-components-select-small-min-width: 5.5rem;
         --amplify-components-select-large-min-width: 7.5rem;
+        --amplify-components-selectfield-border-color: var(--amplify-components-fieldcontrol-border-color);
+        --amplify-components-selectfield-color: var(--amplify-components-fieldcontrol-color);
         --amplify-components-selectfield-flex-direction: column;
+        --amplify-components-selectfield-font-size: var(--amplify-components-fieldcontrol-font-size);
+        --amplify-components-selectfield-focus-border-color: var(--amplify-components-fieldcontrol-focus-border-color);
+        --amplify-components-selectfield-label-color: var(--amplify-components-field-label-color);
```

Comment:
The amplify-label component uses the field label color, this can be seen https://github.com/aws-amplify/amplify-ui/blob/main/packages/ui/src/theme/css/component/field.scss with this rule

```
.amplify-label {
  color: var(--amplify-components-field-label-color);
}
```